### PR TITLE
Cea global constants file creation

### PR DESCRIPTION
**Pull request by BhargavaKrishnaSreepathi**
As the name suggests, i have moved the constants present in global variables to constants file in the respective folders. Also decreased the dependence on `gv` in these files. As there are files where multiple CP values were being used, i.e cp of 4.185 kJ/kgK vs cp of 4185 J/kgK. I changed them to a uniform value of 4185 and adopted the code to reflect the same.

For the reviewers, please provide a different set of eyes regarding the same, to ensure the calculations are not off by a factor of 1000.

### CODE REVIEW

**Comment by BhargavaKrishnaSreepathi** [comment]
@gabriel-happle: please look at this PR and ensure that I am not messing up with your results. Thanks.

File: cea/technologies/thermal_network/thermal_network_matrix.py [hunk 1]
```diff
@@ -2277,7 +2278,7 @@ def calc_aggregated_heat_conduction_coefficient(mass_flow, locator, edge_df, pip
     conductivity_pipe = material_properties.ix['Steel', 'lamda_WmK']  # _[A. Kecebas et al., 2011]
     conductivity_insulation = material_properties.ix['PUR', 'lamda_WmK']  # _[A. Kecebas et al., 2011]
     conductivity_ground = material_properties.ix['Soil', 'lamda_WmK']  # _[A. Kecebas et al., 2011]
-    network_depth = constants.NetworkDepth  # [m]
+    network_depth = NetworkDepth  # [m]
```
Comment (JIMENOFONSECA) [inline cea/technologies/thermal_network/thermal_network_matrix.py]: ....CAPITAL LETTERS

File: cea/technologies/sewage_heat_exchanger.py [hunk 2]
```diff
@@ -150,8 +142,8 @@ def calc_sewageheat( mcp, tin, w_HEX, Vf, cp, h0, min_m, L_HEX, tmin, ATmin):
     city districts. Energy and Buildings.
     """
 
-    mcp_min = min_m * cp # minimum sewage heat capacity in [kW/K]
-    mcp_max = Vf * w_HEX * 0.20 * 1000 * cp   # 20 cm is the depth of the active water in contact with the HEX
+    mcp_min = min_m * (cp/1000) # minimum sewage heat capacity in [kW/K]
```
Comment (JIMENOFONSECA) [inline cea/technologies/sewage_heat_exchanger.py]: ...CAPITAL LETTERS + It is called in some scripts differently, let's keep consistency in the naming

File: cea/technologies/chillers.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from __future__ import division
 import pandas as pd
 from math import log
-from cea.optimization.constants import *
+from cea.optimization.constants import VCC_tcoolin
```
Comment (JIMENOFONSECA) [inline cea/technologies/chillers.py]: ...CAPITAL LETTERS

File: cea/technologies/boilers.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 from scipy.interpolate import interp1d
 from math import log
 import pandas as pd
-from cea.optimization.constants import *
+from cea.optimization.constants import Boiler_P_aux
```
Comment (JIMENOFONSECA) [inline cea/technologies/boilers.py]: ...CAPITAL LETTERS

File: cea/optimization/supportFn.py [hunk 5]
```diff
@@ -8,13 +8,10 @@
...
 import pandas as pd
 import os
-from numpy.random import random_sample
-from pickle import Unpickler
 from deap import base
 from deap import creator
 import json
-import math
-from cea.optimization.constants import *
+from cea.optimization.constants import nHeat, nSolar, nHR
```
Comment (JIMENOFONSECA) [inline cea/optimization/supportFn.py]: ...CAPITAL LETTERS

File: cea/optimization/preprocessing/processheat.py [hunk 6]
```diff
@@ -10,10 +10,10 @@
 from __future__ import division
 import pandas as pd
 from cea.technologies import boilers
-from cea.optimization.constants import *
+from cea.optimization.constants import Boiler_eta_hp, Qmargin_Disc, NG_BACKUPBOILER_TO_OIL_STD, NG_BACKUPBOILER_TO_CO2_STD
```
Comment (JIMENOFONSECA) [inline cea/optimization/preprocessing/processheat.py]: ...CAPITAL LETTERS

File: cea/optimization/master/generation.py [hunk 7]
```diff
@@ -8,7 +8,7 @@
 import random
 from numpy.random import random_sample
 from itertools import izip
-from cea.optimization.constants import *
+from cea.optimization.constants import nHeat, nSolar, nHR
```
Comment (JIMENOFONSECA) [inline cea/optimization/master/generation.py]: ...CAPITAL LETTERS

File: cea/optimization/master/crossover.py [hunk 8]
```diff
@@ -7,7 +7,7 @@
 from __future__ import division
 import random
 from deap import base
-from cea.optimization.constants import *
+from cea.optimization.constants import nHeat, nSolar, nHR
```
Comment (JIMENOFONSECA) [inline cea/optimization/master/crossover.py]: ...CAPITAL LETTERS

**Comment by JIMENOFONSECA** [comment]
@BhargavaKrishnaSreepathi I just made some comments.

**Comment by BhargavaKrishnaSreepathi** [comment]
@JIMENOFONSECA: made the corresponding changes

**Comment by JIMENOFONSECA** [comment]
@BhargavaKrishnaSreepathi I cannot see all the requested changes implemented. There are still several constant values written in lowercase.

**Pull request merged**